master/internal/rm/kubernetesrm/kubernetes_resource_manager.go

```diff
@@ -417,6 +417,18 @@ func (k *ResourceManager) DeleteNamespace(namespaceName string) error {
 	return nil
 }
 
+// RemoveEmptyNamespace removes a namespace from our interfaces in cluster if it is no
+// longer used by any workspace.
+func (k *ResourceManager) RemoveEmptyNamespace(namespaceName string,
+	clusterName string,
+) error {
+	err := k.jobsService.RemoveEmptyNamespace(namespaceName, clusterName)
+	if err != nil {
+		return fmt.Errorf("error removing namespace %s: %w", namespaceName, err)
```

Comment:
I don't htink this err msg is consistent with the other types of RMs

Reply:
Removed the extra err message in agent RM and kept it as rmerrors.ErrNotSupported